src/test/java/worktypelightning/CreateWorkTypeTest.java

```diff
@@ -0,0 +1,39 @@
+package worktypelightning;
+
+import base.BaseTest;
+import salesforce.PageTransporter;
+import salesforce.lightning.WorkTypeLightningPage;
+import salesforce.lightning.WorkTypeForm;
+import salesforce.lightning.WorkTypeInfo;
+import static org.testng.Assert.assertEquals;
+
+public class CreateWorkTypeTest extends BaseTest {
+
+    //@Test
```

Comment:
should delete commented code if its not necessary

Reply:
solved